<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1066</width>
    <height>756</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <layout class="QGridLayout" name="gridLayout_13">
     <item row="4" column="2">
      <spacer name="horizontalSpacer_20">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="4" column="3">
      <widget class="QPushButton" name="delete_damage">
       <property name="text">
        <string>Remove Damage</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <spacer name="horizontalSpacer_24">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Fixed</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="2" rowspan="2" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_21">
       <item>
        <widget class="QLabel" name="label_50">
         <property name="font">
          <font>
           <pointsize>24</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Move</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="move_name">
         <property name="font">
          <font>
           <pointsize>24</pointsize>
          </font>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="2" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_20">
       <item>
        <layout class="QGridLayout" name="gridLayout_9">
         <item row="1" column="0">
          <widget class="QLabel" name="label_62">
           <property name="text">
            <string>Amount of die</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QCheckBox" name="move_1">
           <property name="text">
            <string>+ Move</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="die_at_1"/>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="die_type_1"/>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_65">
           <property name="text">
            <string>Damage at Level 1</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_63">
           <property name="text">
            <string>Type of die</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="times_1"/>
         </item>
         <item row="4" column="1">
          <widget class="QCheckBox" name="level_1">
           <property name="text">
            <string>+ Level</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_70">
           <property name="text">
            <string>Multi Hit</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="1" column="0">
          <widget class="QLabel" name="label_60">
           <property name="text">
            <string>Amount of die</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="die_type_5"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="die_at_5"/>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_58">
           <property name="text">
            <string>Damage at Level 5</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QCheckBox" name="move_5">
           <property name="text">
            <string>+ Move</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_61">
           <property name="text">
            <string>Type of die</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QCheckBox" name="level_5">
           <property name="text">
            <string>+ Level</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_72">
           <property name="text">
            <string>Multi Hit</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLineEdit" name="times_5"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_11">
         <item row="1" column="0">
          <widget class="QLabel" name="label_67">
           <property name="text">
            <string>Amount of die</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_68">
           <property name="text">
            <string>Damage at Level 10</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="die_at_10"/>
         </item>
         <item row="4" column="1">
          <widget class="QCheckBox" name="level_10">
           <property name="text">
            <string>+ Level</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="die_type_10"/>
         </item>
         <item row="3" column="1">
          <widget class="QCheckBox" name="move_10">
           <property name="text">
            <string>+ Move</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_69">
           <property name="text">
            <string>Type of die</string>
           </property>
          </widget>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_74">
           <property name="text">
            <string>Multi Hit</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="times_10"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QGridLayout" name="gridLayout_10">
         <item row="4" column="1">
          <widget class="QCheckBox" name="level_17">
           <property name="text">
            <string>+ Level</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_64">
           <property name="text">
            <string>Amount of die</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="die_type_17"/>
         </item>
         <item row="1" column="1">
          <widget class="QLineEdit" name="die_at_17"/>
         </item>
         <item row="6" column="0">
          <widget class="QLabel" name="label_73">
           <property name="text">
            <string>Multi Hit</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QCheckBox" name="move_17">
           <property name="text">
            <string>+ Move</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_66">
           <property name="text">
            <string>Type of die</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0" colspan="2">
          <widget class="QLabel" name="label_59">
           <property name="text">
            <string>Damage at Level 17</string>
           </property>
          </widget>
         </item>
         <item row="6" column="1">
          <widget class="QLineEdit" name="times_17"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item row="2" column="0" rowspan="3">
      <widget class="QListWidget" name="list_moves">
       <property name="minimumSize">
        <size>
         <width>175</width>
         <height>0</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>200</width>
         <height>16777215</height>
        </size>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_57">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="text">
        <string>List of extra Moves</string>
       </property>
      </widget>
     </item>
     <item row="0" column="0">
      <spacer name="verticalSpacer_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeType">
        <enum>QSizePolicy::Minimum</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>0</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="2" column="2" colspan="2">
      <layout class="QGridLayout" name="gridLayout_12">
       <item row="1" column="1">
        <widget class="QLabel" name="label_49">
         <property name="text">
          <string>Type</string>
         </property>
        </widget>
       </item>
       <item row="5" column="2" colspan="2">
        <widget class="QLineEdit" name="move_casting_time">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="7" column="1">
        <widget class="QLabel" name="label_54">
         <property name="text">
          <string>Range</string>
         </property>
        </widget>
       </item>
       <item row="6" column="1">
        <widget class="QLabel" name="label_56">
         <property name="text">
          <string>Duration</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" colspan="2">
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <item>
          <widget class="QComboBox" name="move_power1"/>
         </item>
         <item>
          <widget class="QComboBox" name="move_power2"/>
         </item>
         <item>
          <widget class="QComboBox" name="move_power3"/>
         </item>
        </layout>
       </item>
       <item row="3" column="1">
        <widget class="QLabel" name="label_55">
         <property name="text">
          <string>Save</string>
         </property>
        </widget>
       </item>
       <item row="9" column="4">
        <spacer name="horizontalSpacer_25">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Preferred</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="2" colspan="2">
        <widget class="QLineEdit" name="move_range"/>
       </item>
       <item row="5" column="1">
        <widget class="QLabel" name="label_71">
         <property name="text">
          <string>Casting Time</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLabel" name="label_51">
         <property name="text">
          <string>Move Power</string>
         </property>
        </widget>
       </item>
       <item row="6" column="5">
        <widget class="QTextBrowser" name="textBrowser_2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.SF NS Text'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:600;&quot;&gt;Common durations:&lt;/span&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt; Instantaneous, 1 minute, &amp;quot;1 minute, Concentration&amp;quot;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="5" column="5">
        <widget class="QTextBrowser" name="textBrowser_3">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.SF NS Text'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:600;&quot;&gt;Common casting times:&lt;/span&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt; 1 action, 1 bonus action, &amp;quot;1 action, recharge&amp;quot;&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2">
        <widget class="QComboBox" name="move_save"/>
       </item>
       <item row="9" column="1" colspan="3">
        <widget class="QTextEdit" name="move_entry"/>
       </item>
       <item row="1" column="2">
        <widget class="QComboBox" name="move_type"/>
       </item>
       <item row="4" column="1">
        <widget class="QLabel" name="label_52">
         <property name="text">
          <string>PP</string>
         </property>
        </widget>
       </item>
       <item row="8" column="1" colspan="2">
        <widget class="QLabel" name="label_53">
         <property name="text">
          <string>Move Description</string>
         </property>
        </widget>
       </item>
       <item row="4" column="2">
        <widget class="QLineEdit" name="move_pp">
         <property name="maximumSize">
          <size>
           <width>50</width>
           <height>16777215</height>
          </size>
         </property>
        </widget>
       </item>
       <item row="7" column="5">
        <widget class="QTextBrowser" name="textBrowser">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>30</height>
          </size>
         </property>
         <property name="html">
          <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'.SF NS Text'; font-size:13pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;p style=&quot; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt; font-weight:600;&quot;&gt;Common ranges&lt;/span&gt;&lt;span style=&quot; font-family:'MS Shell Dlg 2'; font-size:8.25pt;&quot;&gt;: Melee, Self, 100ft, 30ft radius, 10ft cone&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
         </property>
        </widget>
       </item>
       <item row="6" column="2" colspan="2">
        <widget class="QLineEdit" name="move_duration">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>move_name</tabstop>
  <tabstop>move_type</tabstop>
  <tabstop>move_power1</tabstop>
  <tabstop>move_power2</tabstop>
  <tabstop>move_power3</tabstop>
  <tabstop>move_save</tabstop>
  <tabstop>move_pp</tabstop>
  <tabstop>move_casting_time</tabstop>
  <tabstop>move_duration</tabstop>
  <tabstop>move_range</tabstop>
  <tabstop>move_entry</tabstop>
  <tabstop>die_at_1</tabstop>
  <tabstop>die_type_1</tabstop>
  <tabstop>move_1</tabstop>
  <tabstop>level_1</tabstop>
  <tabstop>times_1</tabstop>
  <tabstop>die_at_5</tabstop>
  <tabstop>die_type_5</tabstop>
  <tabstop>move_5</tabstop>
  <tabstop>level_5</tabstop>
  <tabstop>times_5</tabstop>
  <tabstop>die_at_10</tabstop>
  <tabstop>die_type_10</tabstop>
  <tabstop>move_10</tabstop>
  <tabstop>level_10</tabstop>
  <tabstop>times_10</tabstop>
  <tabstop>die_at_17</tabstop>
  <tabstop>die_type_17</tabstop>
  <tabstop>move_17</tabstop>
  <tabstop>level_17</tabstop>
  <tabstop>times_17</tabstop>
  <tabstop>textBrowser_3</tabstop>
  <tabstop>textBrowser_2</tabstop>
  <tabstop>textBrowser</tabstop>
  <tabstop>list_moves</tabstop>
  <tabstop>delete_damage</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
